model/src/main/java/org/jellyfin/apiclient/model/dlna/DeviceProfile.java

```diff
@@ -517,7 +517,7 @@ public final TranscodingProfile GetVideoTranscodingProfile(String container, Str
 			}
 
 			String tempVar = i.getVideoCodec();
-			if (!StringHelper.equalsIgnoreCase(videoCodec, (tempVar != null) ? tempVar : ""))
+			if(!videoCodec.equalsIgnoreCase((tempVar != null) ? tempVar : ""))
```

Comment:
This is no longer null safe and I don't see any checks to prevent `videoCodec` being null here.